app/src/main/java/com/example/yams/GameCreationActivity.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.yams
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+
+class GameCreationActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_game_creation)
+
+        val buttonHomepage: Button = findViewById (R.id.button_GameCreation2Homepage)
```

Comment:
Pour les id on va faire du snake_case. Pour les variables Kotlin du CamelCase. 
Après on peut faire comme on veut mais le mélange `button_GameCreation` c'est interdit.

```suggestion
        val buttonHomepage: Button = findViewById (R.id.button_game_creation_to_homepage)
```

Et puis moi j'aime pas trop les 2 a la place du "to".